API.md

```diff
@@ -1332,6 +1334,10 @@ ImageNotFound means the image could not be found by the provided name or ID in l
 ### <a name="NoContainerRunning"></a>type NoContainerRunning
 
 NoContainerRunning means none of the containers requested are running in a command that requires a running container.
+### <a name="NoContainersInPod"></a>type NoContainersInPod
+
+NoContainersInPod means a pod has no containers on which to perform operation. It contains
```

Comment:
nit.  suggest "perform the operation" or perhaps "perform an operation".

Reply:
Fixed in varlink file below.